src/app/shared/message-table/value.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { MainfluxMsg } from 'app/common/interfaces/mainflux.interface';
+
+@Pipe({name: 'value'})
```

Comment:
Good  idea this piepe, but after to  think about this should probably go in `commons` dir. `shared` is for shared components. 
You could also rename this `msgValue` to be less ambiguous.